<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Admin Page</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>C:/Users/hp/Downloads/admin.png</normaloff>C:/Users/hp/Downloads/admin.png</iconset>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>280</width>
     <height>520</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.0248052, y1:0.011, x2:0.955039, y2:0.926, stop:0 rgba(209, 104, 165, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Serif</family>
     <pointsize>32</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Admin</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>230</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgb(236, 150, 243);}
QPushButton:hover{
	color: rgb(255, 255, 255);}
</string>
   </property>
   <property name="text">
    <string>Create New Quiz</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>230</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgb(236, 150, 243);}
QPushButton:hover{
	color: rgb(255, 255, 255);}
</string>
   </property>
   <property name="text">
    <string>Question Bank</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>230</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgb(236, 150, 243);}
QPushButton:hover{
	color: rgb(255, 255, 255);}
</string>
   </property>
   <property name="text">
    <string>Score History</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>230</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgb(236, 150, 243);}
QPushButton:hover{
	color: rgb(255, 255, 255);}
</string>
   </property>
   <property name="text">
    <string>Change Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>430</y>
     <width>230</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgb(236, 150, 243);}
QPushButton:hover{
	color: rgb(255, 255, 255);}
</string>
   </property>
   <property name="text">
    <string>Sign Out</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>230</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgb(236, 150, 243);}
QPushButton:hover{
	color: rgb(255, 255, 255);}
</string>
   </property>
   <property name="text">
    <string>All users</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
